include/ginkgo/core/matrix/sparsity.hpp

```diff
@@ -167,7 +190,8 @@ class Sparsity : public EnableLinOp<Sparsity<ValueType, IndexType>>,
         : EnableLinOp<Sparsity>(exec, size),
```

Comment:
It would be great that add a constructor to set the value.
It can also be used for user's defined value for user defined value.
```
Sparsity(exec, value)
Sparsity(exec, size={}, num_nonzeros={}, value=1)
```

Reply:
Actually, I partially implemented it in the other PR #350 (https://github.com/ginkgo-project/ginkgo/pull/350/files#diff-b2ea0d623fddd9925021ed16d76f462a), but I will move it here. Thanks.